packages/css_colors/README.md

```diff
@@ -8,13 +8,15 @@ from `dart:ui`, which means they're useful for Flutter apps.
 Import
```

Comment:
The entire "Import" section, including the excerpt, can be removed; the installation tab on pub.dev has that information now, so we are removing it from READMEs.